package.json

```diff
@@ -3,19 +3,29 @@
   "version": "6.5.9",
   "description": "Most modern mobile touch slider and framework with hardware accelerated transitions",
   "scripts": {
-    "build:dev": "cross-env NODE_ENV=development node scripts/build",
-    "build:prod": "cross-env NODE_ENV=production node scripts/build",
-    "build:prod:core": "cross-env NODE_ENV=production node scripts/build --only core",
-    "build:prod:bundle": "cross-env NODE_ENV=production node scripts/build --only bundle",
+    "build:dev": "node scripts/build",
+    "build:dev:core": "node scripts/build --only core",
+    "build:dev:bundle": "node scripts/build --only bundle",
+    "build:dev:svelte": "node scripts/build --only svelte",
+    "build:dev:angular": "node scripts/build --only angular",
+    "build:dev:react": "node scripts/build --only react",
+    "build:dev:vue": "node scripts/build --only vue",
+    "build:prod": "node scripts/build --prod",
+    "build:prod:core": "npm run build:dev:core -- --prod",
+    "build:prod:bundle": "npm run build:dev:bundle -- --prod",
+    "build:prod:svelte": "npm run build:dev:svelte -- --prod",
+    "build:prod:angular": "npm run build:dev:angular -- --prod",
+    "build:prod:react": "npm run build:dev:react -- --prod",
+    "build:prod:vue": "npm run build:dev:vue -- --prod",
     "build-icons-font": "python ./scripts/icon-font/generate.py",
-    "playground:dev": "cross-env NODE_ENV=development node scripts/playground",
-    "playground:prod": "cross-env NODE_ENV=production node scripts/playground",
-    "watch:dev": "cross-env NODE_ENV=development node scripts/watch",
-    "watch:prod": "cross-env NODE_ENV=production node scripts/watch",
-    "core": "npm run build:dev && npm run playground:dev && concurrently --kill-others \"vite ./playground/core\" \"npm run watch:dev\" ",
-    "react": "npm run build:dev && concurrently --kill-others \"vite ./playground/react\" \"npm run watch:dev\"",
-    "vue": "npm run build:dev && concurrently --kill-others \"vite ./playground/vue\" \"npm run watch:dev\"",
-    "svelte": "npm run build:dev && concurrently --kill-others \"vite ./playground/svelte\" \"npm run watch:dev\"",
+    "playground:dev": "node scripts/playground",
+    "playground:prod": "node scripts/playground -- --prod",
+    "watch:dev": "node scripts/watch",
+    "watch:prod": "node scripts/watch -- --prod",
```

Comment:
do we need ` -- ` here? We call node scripts directly here

Reply:
nope, you're right. fixed.